Makefile

```diff
@@ -124,11 +124,8 @@ __check_defined = \
 .PHONY: help
 
 help: ## help on rule's targets
-ifeq ($(IS_WIN),)
```

Comment:
goodbye windows

Reply:
Double check the README and remove any reference to windows as well

Reply:
will check. Windows runs linux anyway